themes/custom/droopler_theme/templates/layout/page.html.twig

```diff
@@ -71,52 +71,58 @@
 {% set header_layout = node.field_header_layout.value %}
```

Comment:
Tam niżej dorzuciłem koment, który dotyczy też tego "zastanego" kodu - żeby to wrzucić do preprocessa. Ale nie upieram się :-)